src/Physics/Plugins/ammoJSPlugin.ts

```diff
@@ -161,6 +166,12 @@ export class AmmoJSPlugin implements IPhysicsEnginePlugin {
     // Ammo's contactTest and contactPairTest take a callback that runs synchronously, wrap them so that they are easier to consume
     private _isImpostorInContact(impostor: PhysicsImpostor) {
         this._tmpContactCallbackResult = false;
+        let group = impostor.getParam("group");
+        let mask = impostor.getParam("mask");
+        if (group && mask) {
+            this._tmpAmmoConcreteContactResultCallback.set_m_collisionFilterGroup(group);
+            this._tmpAmmoConcreteContactResultCallback.set_m_collisionFilterMask(mask);
+        }

```

Comment:
I copy and pasted this snippet from @CedricGuillemet, so this now appears 3 times in this file. I'm unsure where we should refactor this if we didn't want to repeat this snippet

For a future similar fix for `ClosestRayResultCallback`, we may have to repeat this snippet a 4th time if not refactored